introduction.md

```diff
@@ -0,0 +1 @@
+Hi my name is Alex Reith and I'm taking a course on GitHub from Code School.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.